CONTRIBUTING.md

```diff
@@ -55,4 +56,23 @@ After installing medaCy for development, make sure that `pytest` is installed. T
 
     This will show log output during tests and allow you to adust logging level for the test file being run. Read the pytest documentation for details.
 
+## Push Changes to Repository
+Make sure that you are in the development branch of medaCy and from there create a branch within development with the command: `git -b <new_branch_name>`.
```

Comment:
@paulsonnt This command isn't quite right and we only tell the user to create a new branch but not to switch to it. Would you mind changing this to `git checkout -b <new_branch_name>` and changing the wording to reflect that?